server/issuers.go

```diff
@@ -159,6 +170,47 @@ func (c *Server) issuerGetAllHandler(w http.ResponseWriter, r *http.Request) *ha
 	return nil
 }
 
+// issuerV3CreateHandler - creation of a time aware issuer
+func (c *Server) issuerV3CreateHandler(w http.ResponseWriter, r *http.Request) *handlers.AppError {
+	log := lg.Log(r.Context())
+
+	decoder := json.NewDecoder(http.MaxBytesReader(w, r.Body, maxRequestSize))
+	var req issuerV3CreateRequest
+	if err := decoder.Decode(&req); err != nil {
+		c.Logger.Error("Could not parse the request body")
+		return handlers.WrapError(err, "Could not parse the request body", 400)
+	}
+
+	if req.ExpiresAt != nil {
+		if req.ExpiresAt.Before(time.Now()) {
+			c.Logger.Error("Expiration time has past")
+			return &handlers.AppError{
+				Message: "Expiration time has past",
+				Code:    400,
+			}
+		}
+	}
+
+	if err := c.createV3Issuer(Issuer{
```

Comment:
Should the overlap also be added here

Reply:
ValidFrom is part of the issuerV3CreateRequest but not used here should it be added